crates/core/src/registry.rs

```diff
@@ -198,18 +199,20 @@ pub async fn get_all_versions_descending(dependency_name: &str) -> Result<Versio
 
     match revision
         .data
-        .iter()
+        .par_iter()
         .map(|r| Version::parse(&r.version))
         .collect::<std::result::Result<Vec<Version>, _>>()
     {
         Ok(mut versions) => {
             // all versions are semver compliant
-            versions.sort_unstable_by(|a, b| b.cmp(a)); // sort in descending order
+            versions.par_sort_unstable_by(|a, b| b.cmp(a)); // sort in descending order
             Ok(Versions::Semver(versions))
         }
         Err(_) => {
             // not all versions are semver compliant, do not sort (use API sort order)
-            Ok(Versions::NonSemver(revision.data.iter().map(|r| r.version.to_string()).collect()))
+            Ok(Versions::NonSemver(
+                revision.data.par_iter().map(|r| r.version.to_string()).collect(),
+            ))
         }
```

Comment:
Since all the other benchmarks show no benefit from rayon, I suspect this one won't either be beneficial.